Update Listening to the protocol documentation with CDP editor (#210)
ChromeDevTools/debugger-protocol-viewer: pages/index.md

<p>This is especially handy to understand how the DevTools frontend makes use of the protocol.
You can view all requests/responses and methods as they happen.

> <p>You can also open a CDP editor to by clicking on the icon on the left of the input bar : 

Reviewer: did you mean to add this line?

<figure class="screenshot">

>   <a href="images/protocol-monitor.png" target="_blank" style="text-align: center; display:block;">

Reviewer: let's update the existing screenshot?

<code>Page.captureScreenshot</code>. If the command requires parameters, provide them as JSON, for example,
<code>{"command":"Page.captureScreenshot","parameters":{"format": "jpeg"}}</code>.

> <p> You can also open a CDP editor by clicking on the icon on the left of the input bar. This will automatically open an editor which will enable you to better edit your commands. First, enter the command you want to send. The corresponding parameters will then appear and you will just have to edit them and either press CTRL + Enter or click the send icon to send your command.

Reviewer: ```suggestion
<p> By clicking on the icon next to the command input (available starting from Chrome version 117.0.5936.0), you can open the command editor. After you select a CDP command, the editor creates a structured form based on the protocol definitions that allows you to edit the parameters, and see the documentation and types for specific parameters. Send the commands by clicking on the send button or using <code>Ctrl + Enter</code>.  Use the context menu in the list of previously sent commands to open one of them in the editor.
```

> <p> By clicking on the icon next to the command input (available starting from Chrome version 117.0.5936.0), you can open the command editor. After you select a CDP command, the editor creates a structured form based on the protocol definitions that allows you to edit the parameters, and see the documentation and types for specific parameters. Send the commands by clicking on the send button or using <code>Ctrl + Enter</code>.  Use the context menu in the list of previously sent commands to open one of them in the editor.

Reviewer: ```suggestion
<p> By clicking on the icon next to the command input (available starting from Chrome version 117.0.5936.0), you can open the command editor. After you select a CDP command, the editor creates a structured form based on the protocol definitions that allows you to edit parameters, and view their documentation and types. Send the commands by clicking on the send button or using <code>Ctrl + Enter</code>.  Use the context menu in the list of previously sent commands to open one of them in the editor.
```

Reviewer: the previous suggestion was outdated

<code>Page.captureScreenshot</code>. If the command requires parameters, provide them as JSON, for example,
<code>{"command":"Page.captureScreenshot","parameters":{"format": "jpeg"}}</code>.

<p> You can also open a CDP editor by clicking on the icon on the left of the input bar :

<figure class="screenshot">
  <a href="images/open-editor-screenshot.png" target="_blank" style="text-align: center; display:block;">
    <img src="images/open-editor-screenshot.png" height="300" alt="Screenshot of the Protocol Monitor" loading="lazy">
  </a>
</figure>

<p>This will automatically open the editor which will enable you to better edit your commands. First enter the command you want to send. The corresponding parameters will then appear and you will just have to edit them and either press CTRL + Enter or click the send icon to send your command.

> <figure class="screenshot">

Reviewer: let's just keep one screenshot with the editor open and remove the one with the editor closed
